dftimewolf/lib/logging_utils.py

```diff
@@ -0,0 +1,71 @@
+"""Module providing custom logging formatters."""
+import logging
+import random
+
+def _GenerateColorSequences():
```

Comment:
doesn't this only work for Bash? or at least for Linux terminals? Can we make that somewhere clear?

What about if you run th etool on a non-Linux box?

Reply:
It works on macOS. I haven't tried Windows, but pretty sure WSL supports it. I'll add a note.

Reply:
ack